var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -62,53 +73,82 @@ class Zfp(CMakePackage, CudaPackage):
         "use smaller for finer rate granularity. "
         "Use 8 for H5Z-ZFP filter.",
     )
+
     variant(
-        "strided", default=False, description="Enable strided access for progressive zfp streams"
+        "daz",
+        default=False,
+        description="Denormals are zero: "
+        "Treat denormal-only blocks as containing "
+        "all zeroes",
     )
-    variant("aligned", default=False, description="Enable aligned memory allocation")
-    variant("twoway", default=False, description="Use two-way skew-associative cache")
+
     variant(
         "fasthash",
         default=False,
         description="Use a faster but more collision prone hash function",
     )
+
     variant("profile", default=False, description="Count cache misses")
 
+    variant(
+        "round",
+        default="ZFP_ROUND_NEVER",
+        values=("ZFP_ROUND_NEVER", "ZFP_ROUND_FIRST", "ZFP_ROUND_LAST"),
```

Comment:
If it's easy enough to implement, I would prefer `round=never` over `round=ZFP_ROUND_NEVER` for improved readability and user friendliness.